taxonomies/product/categories.txt

```diff
@@ -209,6 +248,41 @@ de: Haushaltschemikalien
 es: Productos químicos domésticos
 fr: Produits chimiques ménagers
 
+<en: Household chemicals
+en: Bleach
+de: Bleichmittel
+fr: Eau de Javel
+
+<en: Household chemicals
+en: Laundry detergents
+de: Waschmittel
```

Comment:
de:waschmittel already is associated to en:laundry-detergents (categories) - de:waschmittel cannot be mapped to entry en:detergents